docker-entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
could we split the startup scripts for clouseau and couchdb or is any advantage of having one script?

Reply:
Yeah, we wouldn't have to maintain two "entry points".